### ISSUE TITLE
Attention mode 'softmax-xformers' is not available

### ISSUE THREAD

**Issue by KhoaVo** [kijai/ComfyUI-SUPIR#19]
I get the following error.

It says I need Pytorch >= 2.0 but it seems I'm already running 2.1.2

`Attention mode 'softmax-xformers' is not available. Falling back to native attention. This is not a problem in Pytorch >= 2.0. FYI, you are running with PyTorch version 2.1.2+cu121`

**Comment by kijai** [comment]
That's not an error though, just a warning... but currently it does seem that xformers is required.
